app/controllers/submit.js

```diff
@@ -725,6 +740,7 @@ export default Ember.Controller.extend(Analytics, BasicsValidations, NodeActions
 
             node.save()
                 .then(() => {
+                    this.set('node.description', this.get('node.description'));
```

Comment:
Can you explain the purpose of this line? It seems strange to set/mutate a value that you already know. I'm not sure this is necessary and might have side effects.